### PR TITLE
Css cleaning

### PR DESCRIPTION
**Pull request by venturewise**
Test URLs:
- Before: https://main--merative2--hlxsites.hlx.page/
- After: https://css-cleaning--merative2--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -176,7 +176,23 @@ main .section-bottom .columns-2-cols > div >:nth-child(2) p:nth-of-type(3)  {
...
-    main .section-top-grid >div:nth-of-type(2) > div:nth-of-type(2) {
+    .section-top-grid > div:nth-child(1) > div:nth-child(1), p, .article-content-wrapper ul {
+        font-family: var(--sans-serif-font-light);
+        font-size: var(--font-size-21);
+        font-weight: var(--font-weight-light);
+        line-height: var(--line-height-160);
+        letter-spacing: var(--letter-spacing-1);
+        }
+
+    main .section-top-grid > div:nth-of-type(2) > div:nth-of-type(2) > h1, main .section-top-grid > div:nth-of-type(2) > div:nth-of-type(2) > p {
```
Comment (arumsey) [inline blocks/columns/columns.css:187]: Can you add a new line after `,`? When I first saw this I thought it was one giant selector. 😮 There is a lot of trust in the HTML markup here.

File: styles/styles.css [hunk 2]
```diff
@@ -267,23 +267,54 @@ blockquote {
...
 }
 
-main .column-content-top-border div>div {
+main .section > div {
+  width: 80vw;
+  margin: 0 auto;
+  }
+
+main .columns-gap-50 > div {
+  gap: 50px
```
Comment (arumsey) [inline styles/styles.css:276]: `;`

File: styles/styles.css [hunk 3]
```diff
@@ -267,23 +267,54 @@ blockquote {
...
 }
 
-main .columns.columns-gap-50>div {
+/* Section Top and Bottom */
+main .section-top div > h1 {
+  margin: var(--spacer-element-08) 0 var(--spacer-element-07);
+}
+
+main .columns.columns-gap-50 > div {
   gap: 50px
```
Comment (arumsey) [inline styles/styles.css:290]: `;`

File: styles/styles.css [hunk 4]
```diff
@@ -333,16 +364,30 @@ main .article-content-wrapper h2:first-of-type {
...
+main .section .column-content-top-border > div > div h1 {
+  font-size: 96px;
+  line-height: 120%;
+  letter-spacing: 0;
+  font-weight: 300;
+  margin: 0;
+  }
+
+main .columns.row-header div > div > h2,
+main .section.section-header>div:nth-child(1) > h1,
```
Comment (arumsey) [inline styles/styles.css:376]: space between `>`

File: styles/styles.css [hunk 5]
```diff
@@ -333,16 +364,30 @@ main .article-content-wrapper h2:first-of-type {
...
 
+main .section .column-content-top-border > div > div h1 {
+  font-size: 96px;
+  line-height: 120%;
+  letter-spacing: 0;
+  font-weight: 300;
+  margin: 0;
+  }
+
+main .columns.row-header div > div > h2,
```
Comment (arumsey) [inline styles/styles.css:375]: same Q here. Is this specificity required?

File: blocks/columns/columns.css (hunk 1 above)
Comment (venturewise) [reply blocks/columns/columns.css:187]: Agreed on your 'trust' concern. I saw a lot of this, but I'm not close enough to project to know how best to change it (at least not yet).

File: styles/styles.css (hunk 2 above)
Comment (venturewise) [reply styles/styles.css:276]: Nice catch. I am also of the school to always use a semicolon, even on the last line. There are probably more of these from the existing code.

File: blocks/columns/columns.css (hunk 1 above)
Comment (arumsey) [reply blocks/columns/columns.css:187]: Makes sense. No sense rocking the boat too much.

**Review by arumsey — APPROVED** [review]
All I could find were a few formatting issues.

File: styles/styles.css [hunk 6]
```diff
@@ -333,16 +364,30 @@ main .article-content-wrapper h2:first-of-type {
   margin: unset;
 }
 
+main .section .column-content-top-border > div > div h1 {
```
[the remark this answers is not in the document]
Comment (venturewise) [reply styles/styles.css:367]: I don't know. I haven't found where .column-content-top-border is used yet (not my code).

File: styles/styles.css (hunk 5 above)
Comment (venturewise) [reply styles/styles.css:375]: Same answer -- I have not run into its use yet.

**Comment by venturewise** [comment]
I will add the changes in my next PR (as I've already started a new branch).

**Pull request merged**